{"name": "test_auth[merskii_lexa@bk.ru-wertuoz1988]", "status": "failed", "statusDetails": {"message": "AssertionError: assert not True\n +  where True = <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>>()\n +    where <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>> = <pages.elements.WebElement object at 0x000002337EBF3DC8>.is_visible\n +      where <pages.elements.WebElement object at 0x000002337EBF3DC8> = <pages.pages_auth.RTmis object at 0x000002337F7536C8>.error", "trace": "web_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"001bdab4f049788637f117e94fd31c32\")>, login = 'merskii_lexa@bk.ru', password = 'wertuoz1988'\n\n    @allure.feature('Авторизация на https://rtmis.ru/')\n    @allure.story('Тест с авторизации с рандомизируемыми данными')\n    def test_auth(web_browser, login:str, password:str):\n        web = RTmis(web_browser)\n    \n        with allure.step(f\"Открываем {web.url}\"):\n          web.get(url=web.url)\n        with allure.step(\"Заполняем форму (логин:пароль)\"):\n          assert web.login.send_keys(login)\n          assert web.password.send_keys(password)\n        with allure.step(\"Нажимаем на кнопку войти\"):\n          web.auth.human_click()\n        with allure.step(\"Определям на странице сообщение об ошибке авторизации\"):\n>           assert not web.error.is_visible()\nE           assert not True\nE            +  where True = <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>>()\nE            +    where <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>> = <pages.elements.WebElement object at 0x000002337EBF3DC8>.is_visible\nE            +      where <pages.elements.WebElement object at 0x000002337EBF3DC8> = <pages.pages_auth.RTmis object at 0x000002337F7536C8>.error\n\ntests\\test_auth.py:18: AssertionError"}, "steps": [{"name": "Открываем https://demo.rtmis.ru/", "status": "passed", "start": 1643733511636, "stop": 1643733515947}, {"name": "Заполняем форму (логин:пароль)", "status": "passed", "start": 1643733515947, "stop": 1643733516074}, {"name": "Нажимаем на кнопку войти", "status": "passed", "start": 1643733516074, "stop": 1643733526492}, {"name": "Определям на странице сообщение об ошибке авторизации", "status": "failed", "statusDetails": {"message": "AssertionError: assert not True\n +  where True = <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>>()\n +    where <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>> = <pages.elements.WebElement object at 0x000002337EBF3DC8>.is_visible\n +      where <pages.elements.WebElement object at 0x000002337EBF3DC8> = <pages.pages_auth.RTmis object at 0x000002337F7536C8>.error\n", "trace": "  File \"C:\\Users\\ole lukoie\\Desktop\\ui\\tests\\test_auth.py\", line 18, in test_auth\n    assert not web.error.is_visible()\n"}, "start": 1643733526492, "stop": 1643733526526}], "parameters": [{"name": "login", "value": "'merskii_lexa@bk.ru'"}, {"name": "password", "value": "'wertuoz1988'"}], "start": 1643733511636, "stop": 1643733526526, "uuid": "1c346e05-0cce-4833-8a6d-a5d5aed65cbd", "historyId": "b816c5af7c911ecae8770473435f7f6b", "testCaseId": "5306df55395c93b83e69abe887b683bd", "fullName": "tests.test_auth#test_auth", "labels": [{"name": "feature", "value": "Авторизация на https://rtmis.ru/"}, {"name": "story", "value": "Тест с авторизации с рандомизируемыми данными"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "host", "value": "DESKTOP-2HIEP5G"}, {"name": "thread", "value": "1520-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}